### DISCUSSION TITLE
Declare a section in the isort config listing the Ansible-land dependencies

### DISCUSSION THREAD

**Discussion by webknjaz** [ansible/ansible-navigator#960] [Linters, linting, formatting, pre-commit, CI]
It'd be nice to have a section declared in the `isort` config that would list the Ansible-land dependencies. As in things from our ecosystem/colleagues. Separately from the rest of the third-party libraries.

Likely depends on: https://github.com/ansible/ansible-navigator/issues/784
Related: https://github.com/ansible/ansible-navigator/issues/785

_Originally posted by @webknjaz in https://github.com/ansible/ansible-navigator/pull/777#discussion_r788735239_

**Comment by cidrblock** [comment]
unassigned self as I do not see a pressing requirement for this to happen until needed, assigning @webknjaz to close or remediate
